release/readme.md

```diff
@@ -1,29 +1,54 @@
 # Steps to follow for each release
 
 ## Finalise opam code for release
-* update version in opam files, configure.ac
+* update version in all the opam files and in configure.ac
 * run `make configure` to regenerate `./configure` [checked by github actions]
 * update copyright headers
-* run `make tests`, `opam-rt` [checked by github actions & appveyor]
-* update the CHANGE file: take `master_changes.md` content to fil it
+* if you're releasing the first final release of a new branch (e.g. 2.2.0): make sure `root_version` in OpamFile.ml is set to the final release number (e.g. for 2.2.0, root_version should be 2.2). Make sure that opamFormatUpgrade.ml also contains an upgrade function from the previous version (that function will most likely be empty)
+* run `make tests`, `opam-rt` [checked by github actions]
+* update the CHANGE file: take `master_changes.md` content to fill it
 
 ## Github release
 
 [ once bump version & changes PRs merged ]
-* tag the release (git tag -a 2.2.0; git push origin 2.2.0)
-* /!\ Once the tag pushed, it can be updated only in case of severe issue
+* tag the release (git tag -am 2.2.0 2.2.0; git push origin 2.2.0)
+* /!\ Once the tag pushed, it can be updated [different commit] only in case of severe issue
 * create a release (or prerelease if intermediate release) draft on github based on your tag (https://github.com/ocaml/opam/releases/new)
-* generate opam artifacts, using `shell/release.sh`, it requires to have Docker install with several remotes, the different arches
-* add releases notes (content of `master_changes.md`) in the release
-* upload signature of artefacts
+* fetch locally the tag
```

Comment:
It's missing the readme update, otherwise, lgtm

Reply:
done